### DISCUSSION TITLE
你好有群吗

### DISCUSSION THREAD

**Discussion by whooc** [ArtalkJS/Artalk#91] [Ideas]

**Comment by qwqcode** [comment]
由于时间不太充裕，现在还暂时没有建群哦，感谢您对 Artalk 的关注。

**Comment by monsterxcn** [comment]
项目相关话题可以在 [Discussions](https://github.com/ArtalkJS/Artalk/discussions) 讨论！

**Comment by RubberTale** [comment]
这里讨论不是很好吗？为何还需要群啊？

**Reply by whooc** [reply]
Such a long reply
